packages/amplify-util-uibuilder/commands/utils/shouldRenderComponents.js

```diff
@@ -1,32 +1,31 @@
-const logger = require('./logger');
-
+const { printer } = require('amplify-prompts');
 async function shouldRenderComponents(context) {
   if (process.env.FORCE_RENDER) {
-    logger.info('Forcing component render since environment variable flag is set.');
+    printer.debug('Forcing component render since environment variable flag is set.');
```

Comment:
should all of these be debug or warn? debug will only be printed if the cli is run with `--debug`

Reply:
In general I didn't want this plugin to be too chatty, but information like this is very helpful for debugging issues.. So I like --debug